docs/source/get_involved/governance.rst

```diff
@@ -70,6 +70,9 @@ responsibilities, and appointment process in more detail below.
    * - :ref:`cc-members`
      - Conflict resolution, technical leadership, project management
      - Nomination by core developers, vote by core developers, 2/3 majority and simple CC majority
+   * - :ref:`cc-observers`
+     - Full view of CC, direct input on CC decisions
+     - Nomination by core developers, vote by core developers, 2/3 of all cast votes and simple CC majority
```

Comment:
```suggestion
     - Nomination by CC by simple CC majority
```

Reply:
Should it not be: "Nomination by core developers, vote by CC members, simple CC majority"?